src/Input.tsx

```diff
@@ -43,9 +43,11 @@ const Input: React.FunctionComponent<IInput> = React.forwardRef<HTMLInputElement
 
   return (
     <Box
-      px={3}
-      py={2}
+      p="10px"
```

Comment:
WHOA. What happened to using the styled-system sizes or whatever they're called?

Reply:
The gaps are too large, we don't have a sizing that maps to 10px right now

Reply:
In general right now they aren't very practical until we remap them
1=> 4px
2=> 8px
3=> 16px
4=>32px